services/helper/bigBlueButton.js

```diff
@@ -0,0 +1,82 @@
+const bigBlueButtonUrl = process.env.BIG_BLUE_BUTTON_URL + process.env.BIB_BLUE_BUTTON_BASE_URL;
+const crypto = require("crypto");
+const request = require("../../generics/requests");
+const endpoints = require('../../constants/endpoints');
+
+module.exports = class SessionsHelper {
+    static async createMeeting(meetingId,meetingName,attendeePW,moderatorPW) {
+        try {
+
+            // let recordingCallBackUrl = encodeURI(process.env.RECORDING_READY_CALLBACK_URL);
+            // "&meta_bbb-recording-ready-url=" + recordingCallBackUrl;
+            let endMeetingCallBackUrl = process.env.MEETING_END_CALLBACK_EVENTS + "%2F" + meetingId;
```

Comment:
@aman-tunerlabs where we have implemented end call back, to know that the session is completed by mentor ?

Reply:
@aman-g202 This is a callback URL that gets called once the meeting is over.

Reply:
@aman-tunerlabs This means this will call our session/completed api ?